packages/sitecore-jss-dev-tools/src/codegen/extract-components.ts

```diff
@@ -0,0 +1,76 @@
+import { ExtractedFileType, resolveComponentImportFiles, sendCode } from './utils';
+import { fetchBearerToken } from '../auth/fetch-bearer-token';
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+
+type ExtractComponentArgs = {
+  appFolder?: string;
```

Comment:
`appFolder` is no longer needed, right? Since this script is expected to be executed at build time within the application itself.

Reply:
appFolder is needed to find tsconfig - and for unit testing

Reply:
for unit tests you need to mock `process.cwd`, this property can be removed